packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -36,6 +36,7 @@ class CameraValue {
     this.isTakingPicture,
     this.isStreamingImages,
     bool isRecordingPaused,
+    this.imageStreamImageFormat,
```

Comment:
Don't need this in CameraValue as CameraController also contains the imageFormat